libraries/AP_MotionController/AP_MotionController.cpp

```diff
@@ -15,22 +15,22 @@
     serial bus and can be managed by the library. Each roboclaw can control
     two motors, therefore, eight motors in total. 
  */
-#include <AP_MotionController/AP_MotionController.h>
+#include <GCS_MAVLink/GCS.h>
+#include "AP_MotionController.h"
 
 extern const AP_HAL::HAL &hal; // External reference for console debugging
 
 const AP_Param::GroupInfo AP_MotionController::var_info[] = {
     // @Param: MASK
-    // @DisplayName: MOCR_MASK
+    // @DisplayName: SERVO_MOCR_MASK
```

Comment:
The Display name shouldn't be the same as the parameter name.  I'd say it should probably be something like "RoboClaw motion controller board enable mask" or something to that effect.